compiler-plugin/src/main/java/io/ballerina/stdlib/tcp/compiler/PluginConstants.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.tcp.compiler;
+
+import io.ballerina.compiler.syntax.tree.SyntaxKind;
+
+/**
+ * TCP compiler plugin constants.
+ */
+public class PluginConstants {
+    public static final String TCP_106 = "TCP_106";
+    public static final String TEMPLATE_CODE_GENERATION_HINT = "Template generation for empty service";
+    public static final String MODULE_PREFIX = "ballerina/tcp" + SyntaxKind.COLON_TOKEN.stringValue();
+
+    // expected parameters and return types
+    public static final String READONLY_INTERSECTION = "readonly & ";
+    public static final String CALLER = "Caller";
+    public static final String BYTE_ARRAY = "byte[]";
+    public static final String ERROR = "Error";
+    public static final String OPTIONAL = "?";
+    public static final String NIL = "()";
+
+    /**
+     * Compilation Errors of TCP module.
+     */
+    public enum CompilationErrors {
+        REMOTE_KEYWORD_EXPECTED_IN_0_FUNCTION_SIGNATURE("`remote` keyword expected in `{0}` function signature.",
+                "TCP_101"),
+        SERVICE_DOES_NOT_CONTAIN_ON_BYTES_FUNCTION("Service does not contain `onBytes` function", "TCP_102"),
+        NO_PARAMETER_PROVIDED_FOR_0_FUNCTION_EXPECTS_1_AS_A_PARAMETER("No parameter provided for `{0}`, " +
+                "function expects `{1}` as a parameter", "TCP_104"),
+        INVALID_PARAMETER_0_PROVIDED_FOR_1_FUNCTION_EXPECTS_2("Invalid parameter `{0}` provided for " +
+                "`{1}`, function expects `{2}`", "TCP_105"),
+        INVALID_PARAMETER_0_PROVIDED_FOR_1_FUNCTION("Invalid parameter `{0}` provided for `{1}` function", "TCP_103"),
+        INVALID_RETURN_TYPE_0_FUNCTION_1_RETURN_TYPE_SHOULD_BE_A_SUBTYPE_OF_2("Invalid return type `{0}` " +
+                "provided for function `{1}`, return type should be a subtype of `{2}`", "TCP_106"),
```

Comment:
Shall we use the defined constant for the error code here ?

Reply:
I've mistakenly used the same error code "TCP_106". So changed that to a new error code.

Reply:
Seems like we have completely removed the 106 error code (now the list has 101-110 but without 106). Do we need to re-number ?

Reply:
We have it here, https://github.com/ballerina-platform/module-ballerina-tcp/pull/756/files#diff-345480b529cd426e8090506e70d5713c812658a918f6d3d6ea0469f8b239971aR27 as a Hint, not as a compilation error.